domains/client/domain-operator/src/tests.rs

```diff
@@ -1619,6 +1624,11 @@ async fn test_true_invalid_bundles_illegal_extrinsic_proof_creation_and_verifica
         bundle_extrinsic_root =
             BlakeTwo256::ordered_trie_root(extrinsics.clone(), StateVersion::V1);
         opaque_bundle.sealed_header.header.bundle_extrinsics_root = bundle_extrinsic_root;
+        opaque_bundle.sealed_header.header.bundle_size = opaque_bundle
```

Comment:
maybe introducing `size()` on `opaque_bundle` is a good idea

Reply:
Seems we already have a `size()` for returning the `bundle_size` field, the field may not be necessary since the consensus node can't trust this field and has to recalculate the size during `validate_bundle`, perhaps we can remove this field and change `size()` to calculate the size instead, it requires update on the spec cc @dariolina 

Any thoughts?

Reply:
iirc `bundle_size` ER field also doesn't have a fraud proof. If you say it's useless I have no objections to removing it from ER

Reply:
This field is attached to the bundle header, not the ER. The consensus node can perform the bundle size check (ensure the bundle size does not exceed the limit) itself thus we don't need fraud proof.